### PR TITLE
All material type

### PR DESCRIPTION
**Pull request by Karimelsayed98**

### CODE REVIEW

File: packages/backend/src/api/blog/blog.service.ts [hunk 1]
```diff
@@ -118,4 +119,19 @@ export class BlogService extends TypeOrmCrudService<Blog> {
     blog.views += 1;
     await this.repository.save(blog);
   }
+
+  public async getAllBlogs(skip = 0, take = 10): Promise<PaginatedResponse<Blog>> {
+    const [blogs, count] = await this.repository.findAndCount({
+      where: { publishedAt: Not(IsNull()) },
+      relations: ['user', 'category'],
+      order: { createdAt: 'DESC' },
```
Comment (chantalchawki) [inline packages/backend/src/api/blog/blog.service.ts:127]: publishedAt not createdAt

File: packages/backend/src/api/courses/courses.service.ts [hunk 2]
```diff
@@ -201,4 +202,19 @@ export class CoursesService {
 
     return slug;
   }
+
+  public async getAllCourses(skip = 0, take = 10): Promise<PaginatedResponse<Course>> {
+    const [courses, count] = await this.repository.findAndCount({
+      where: { publishedAt: Not(IsNull()) },
+      relations: ['creator', 'category'],
+      order: { createdAt: 'DESC' },
```
Comment (chantalchawki) [inline packages/backend/src/api/courses/courses.service.ts:210]: publishedAt not createdAt

File: packages/blogs-client/gatsby-node.js [hunk 3]
```diff
@@ -4,8 +4,7 @@ const config = require('./config');
 
 exports.createPages = async ({ actions: { createPage } }) => {
   const {data: {data: blogs} }  = await axios.get(`${config.API_URL}/api/v1/blogs`);
-  const publishedBlogs = blogs.filter(blog => blog.url !== null);
-  publishedBlogs.forEach(blog => 
+  blogs.forEach(blog =>
```
Comment (chantalchawki) [inline packages/blogs-client/gatsby-node.js:7]: this will get 10 blogs only

Comment (chantalchawki) [reply packages/blogs-client/gatsby-node.js:7]: suggestion: make the limit a bigger number

File: packages/blogs-client/src/templates/displayPage/index.tsx [hunk 4]
```diff
@@ -75,8 +75,8 @@ const DisplayPage: React.FC<Props> = ({ pageContext: { blog } }: Props) => {
           crossOrigin="anonymous"
         />
       </Helmet>
+      <img className="flex-auto h-64 w-full border-solid" src={blog.thumbnailURL} alt={blog.name} />
```
Comment (chantalchawki) [inline packages/blogs-client/src/templates/displayPage/index.tsx:78]: is this intended or a mistake?

Comment (Karimelsayed98) [reply packages/blogs-client/src/templates/displayPage/index.tsx:78]: @chantalchawki intended to follow the wireframes, the div has max-w-screen-md while the thumbnail should be full width

**Pull request merged**